Carsus_atomic_data/Custom_atomic_data.ipynb

```diff
@@ -0,0 +1,102 @@
+{
```

Comment:
<p>Try to be a bit more intentional with which font styles you're using when. It's a bit odd how this text is small and bold, and different from other text. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/stardis/pull/226/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>